src/FieldWarning/Assets/Camera/SlidingCameraBehaviour.cs

```diff
@@ -274,62 +336,89 @@ private void ClampCameraAltitude()
     {
         _targetPosition.y = Mathf.Clamp(
                 _targetPosition.y,
-                Terrain.activeTerrain.SampleHeight(_targetPosition) + _minAltitude,
+                _terrain.SampleHeight(_targetPosition) + _minAltitude,
                 _maxAltitude);
     }
 
     private void PanFromScreenBorder()
     {
         if ((Input.mousePosition.x <= _borderPanningOffset && Input.mousePosition.x >= 0
-            && Input.mousePosition.y <= _borderPanningCornerSize && Input.mousePosition.y >= 0)
-            || (Input.mousePosition.x <= _borderPanningCornerSize && Input.mousePosition.x >= 0
-            && Input.mousePosition.y <= _borderPanningOffset && Input.mousePosition.y >= 0)) { //Lower-left screen corner
+                && Input.mousePosition.y <= _borderPanningCornerSize
+                && Input.mousePosition.y >= 0)
+                || (Input.mousePosition.x <= _borderPanningCornerSize
+                && Input.mousePosition.x >= 0
+                && Input.mousePosition.y <= _borderPanningOffset && Input.mousePosition.y >= 0)) {
+            // Lower-left screen corner
```

Comment:
Should we separate this conditional logic into bool-returning methods? Like `if (MouseAtScreenBottonLeft())`? Might make this method a little cleaner to read but would also clutter up the class with a bunch of helper functions, so I dunno, what are your thoughts?

Reply:
Resolved with 9946d324dd184379760ab7685c83e29828045d0b